### PR TITLE
Adding a simple changelog.md that can be used as a base

### PR DESCRIPTION
**Pull request by jonathanschroeter**
Created a simple changelog.md that links to the official milestone. Tried to follow [keepachangelog](https://keepachangelog.com/en/1.0.0/) as closely as possible. #81

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+### Added
+- Rust-based lib that has a C FFI.
+- Utility in the lib to receive a Generic SQL query and execute against a relational database server.
+- Support for MySQL and Postgres database servers.
+- Placeholder argument feature
+- Ability to Allow database backend to be dynamically configured (without need to recompile) i.e. ability to issue query to MySQL or   Postgres.
+- A simple Python client example to demonstrate how we can call into datajoint-core
+- Documentation for the project
+- Tests with reasonable coverage
+- A changelog
```
Comment (jackson-nestelroad) [inline changelog.md:18]: Can remove this one, lol

File: changelog.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+## [0.1.0] - 2021-11-15
+### Added
+- Rust-based lib that has a C FFI.
+- Utility in the lib to receive a Generic SQL query and execute against a relational database server.
+- Support for MySQL and Postgres database servers.
+- Placeholder argument feature
+- Ability to Allow database backend to be dynamically configured (without need to recompile) i.e. ability to issue query to MySQL or   Postgres.
+- A simple Python client example to demonstrate how we can call into datajoint-core
+- Documentation for the project
+- Tests with reasonable coverage
```
Comment (jackson-nestelroad) [inline changelog.md:17]: Change to "Initial unit tests and integration tests."

File: changelog.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.1.0] - 2021-11-15
+### Added
+- Rust-based lib that has a C FFI.
+- Utility in the lib to receive a Generic SQL query and execute against a relational database server.
+- Support for MySQL and Postgres database servers.
+- Placeholder argument feature
+- Ability to Allow database backend to be dynamically configured (without need to recompile) i.e. ability to issue query to MySQL or   Postgres.
+- A simple Python client example to demonstrate how we can call into datajoint-core
```
Comment (jackson-nestelroad) [inline changelog.md:15]: Reword to remove first-person pronoun

File: changelog.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.1.0] - 2021-11-15
+### Added
+- Rust-based lib that has a C FFI.
+- Utility in the lib to receive a Generic SQL query and execute against a relational database server.
+- Support for MySQL and Postgres database servers.
+- Placeholder argument feature
```
Comment (jackson-nestelroad) [inline changelog.md:13]: "feature" is ambiguous. I might just say "Placeholder arguments" or "Support for placeholder arguments"

File: changelog.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.1.0] - 2021-11-15
+### Added
+- Rust-based lib that has a C FFI.
```
Comment (jackson-nestelroad) [inline changelog.md:10]: Do we want to be more specific on what exactly is in the library? Like connections, cursors, generic value decoding, etc.

**Review by jackson-nestelroad — COMMENTED** [review]
A few minor things to make it a bit cleaner. Also make sure the formatting is consistent for each point (consistent capitalization, period at end of each point, consistent spacing, etc.)

File: changelog.md (hunk 5 above)
Comment (jonathanschroeter) [reply changelog.md]: Just updated this to be more specific. Do you think it looks better like this? or should it return to being more generic?

**Review by Rhijke — APPROVED** [review]

**Review by jackson-nestelroad — APPROVED** [review]

**Pull request merged**